examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6699,6 +6699,115 @@ cluster RadonConcentrationMeasurement = 1071 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+/** Functionality to retrieve operational information about a managed Wi-Fi network. */
```

Comment:
This is a huge pane of functionality. To assist in validating the HRAP device types, strongly recommend making a different sample for HRAP

Reply:
There is a separate sample app already, will add the TBRM cluster to that instead.